src/stories/canvas/groups/default.stories.tsx

```diff
@@ -0,0 +1,134 @@
+import React, { useEffect } from "react";
+
+import type { Meta, StoryFn } from "@storybook/react";
+import groupBy from "lodash/groupBy";
+
+import { BlockGroups, Group } from "../../../components/canvas/groups";
+import { BlockState, Graph, GraphCanvas, GraphState, TBlock, useGraph, useGraphEvent } from "../../../index";
+import { useFn } from "../../../utils/hooks/useFn";
+import { BlockStory } from "../../main/Block";
+
+const createConfig = () => {
+  const blocks: TBlock[] = [
+    {
+      id: "block1",
+      is: "block",
+      name: "Block 1",
+      x: 100,
+      y: 100,
+      width: 200,
+      height: 150,
+      group: "group1",
+      selected: false,
+      anchors: [],
+    },
+    {
+      id: "block2",
+      is: "block",
+      name: "Block 2",
+      x: 150,
+      y: 250,
+      width: 200,
+      height: 150,
+      group: "group1",
+      selected: false,
+      anchors: [],
+    },
+    {
+      id: "block3",
+      is: "block",
+      name: "Block 3",
+      x: 500,
+      y: 100,
+      width: 200,
+      height: 150,
+      group: "group2",
+      selected: false,
+      anchors: [],
+    },
+    {
+      id: "block4",
+      is: "block",
+      name: "Block 4",
+      x: 550,
+      y: 250,
+      width: 200,
+      height: 150,
+      group: "group2",
+      selected: false,
+      anchors: [],
+    },
+    {
+      id: "block5",
+      is: "block",
+      name: "Block without group",
+      x: 300,
+      y: 400,
+      width: 200,
+      height: 150,
+      selected: false,
+      anchors: [],
+    },
+  ];
+
+  return { blocks };
+};
+
+const MyGroup = Group.define({
+  style: {
+    background: "rgba(100, 100, 100, 0.1)",
+    border: "rgba(100, 100, 100, 0.3)",
+    borderWidth: 2,
+    selectedBackground: "rgba(100, 100, 100, 1)",
+    selectedBorder: "rgba(100, 100, 100, 1)",
+  },
+});
+
+const GroupsLayer = BlockGroups.withBlockGrouping({
+  groupingFn: (blocks: BlockState[]) => {
+    return groupBy(blocks, (block) => block.$state.value.group);
+  },
+  mapToGroups: (grounId: string, { rect }) => ({
+    id: grounId,
+    rect,
+    component: MyGroup,
+  }),
+});
+
+const GraphApp = () => {
+  const { graph, setEntities, start } = useGraph({});
+  const config = createConfig();
+
+  useEffect(() => {
+    const layer = graph.addLayer(GroupsLayer, {
+      draggable: true,
```

Comment:
Draggable doesn't work in stories.
This is probably due to the absence of `canChangeBlockGeometry: ECanChangeBlockGeometry.ALL,` on line 99 in the config, which is passed to the useGraph.